From: Wan Jiabing <wanjiabing@vivo.com>
Subject: selftests: vm: remove duplicated include in hugepage-mremap

Fix following checkinclude.pl warning:
./tools/testing/selftests/vm/hugepage-mremap.c: fcntl.h is included
more than once.

Link: https://lkml.kernel.org/r/20211021122944.8857-1-wanjiabing@vivo.com
Signed-off-by: Wan Jiabing <wanjiabing@vivo.com>
Cc: Mina Almasry <almasrymina@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/vm/hugepage-mremap.c |    1 -
 1 file changed, 1 deletion(-)

--- a/tools/testing/selftests/vm/hugepage-mremap.c~mm-hugepages-add-hugetlb-vma-mremap-test-v8-fix
+++ a/tools/testing/selftests/vm/hugepage-mremap.c
@@ -12,7 +12,6 @@
 #include <stdio.h>
 #include <unistd.h>
 #include <sys/mman.h>
-#include <fcntl.h>
 #include <errno.h>
 #include <fcntl.h> /* Definition of O_* constants */
 #include <sys/syscall.h> /* Definition of SYS_* constants */
_
